src/components/DataTable/_data-table.scss

```diff
@@ -1 +1,67 @@
 @import '~carbon-components/scss/components/data-table/data-table';
+
+.table-header-label {
+  &-start {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: start;
+    }
+  }
+
+  &-center {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: center;
+    }
+  }
+
+  &-end {
+    .bx--table-header-label {
+      width: 100%;
+      text-align: end;
+    }
+  }
+}
+
+.bx--data-table td:last-of-type {
+  padding-right: $spacing-09 + $spacing-02;
```

Comment:
Is there a design spec to show why we're deviating from the Carbon values for padding in table cells? I suggest we use the standard values to be in line with Carbon: https://github.com/carbon-design-system/carbon/blob/c87a493f8dd86b5289fda9e59a4ee84639652028/packages/components/src/components/data-table/_data-table-core.scss#L88-L94

Reply:
We are not able to align table column header and its respective data in cells while using standard spacing. i.e for 3.25 rem for which we used $spacing-09 + $spacing-02, but if we use $spacing-09 only then it is not aligning as showing in attached screenshot shared over slack

Reply:
I think we need to just use the `text-align` property here. The padding of these cells should use the standard padding we get from the Carbon component.